corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -100,37 +107,24 @@ def migrate_from_migration(cls, apps, schema_editor):
     def couch_db(cls):
         return couch_config.get_db(cls.couch_db_slug())
 
-    def add_arguments(self, parser):
-        parser.add_argument(
-            '--dry-run',
-            action='store_true',
-            dest='dry_run',
-            default=False,
-            help='Do not actually modify the database, just verbosely log what will happen',
-        )
-
-    def handle(self, dry_run=False, **options):
-        log_prefix = "[DRY RUN] " if dry_run else ""
-
-        logger.info("{}Found {} {} docs and {} {} models".format(
-            log_prefix,
-            get_doc_count_by_type(self.couch_db(), self.couch_doc_type()),
+    def handle(self, **options):
+
+        doc_count = get_doc_count_by_type(self.couch_db(), self.couch_doc_type())
+        logger.info("Found {} {} docs and {} {} models".format(
+            doc_count,
             self.couch_doc_type(),
             self.sql_class().objects.count(),
             self.sql_class().__name__,
         ))
+        doc_index = 0
         for doc in get_all_docs_with_doc_types(self.couch_db(), [self.couch_doc_type()]):
-            logger.info("{}Looking at {} doc with id {}".format(
-                log_prefix,
+            doc_index += 1
+            logger.info("Looking at {} doc #{} of {} with id {}".format(
                 self.couch_doc_type(),
+                doc_index,
+                doc_count,
                 doc["_id"]
             ))
             with transaction.atomic():
                 model, created = self.update_or_create_sql_object(doc)
```

Comment:
This will still create or update (write) every doc returned by `get_all_docs_with_doc_types(...)`, right? Where is the code that only migrates missing documents (during deploy, for example)?

Reply:
That's correct. I'd prefer to do this safer change first, until I understand why the deploy found any unmigrated docs.